src/commands/synthetics/run-tests-lib.ts

```diff
@@ -230,7 +231,7 @@ export const getTestsList = async (
     .map((suite) =>
       suite.content.tests.map((test) => ({
         config: overrideTestConfig(test),
-        id: test.id,
+        id: normalizePublicId(test.id) || '',
```

Comment:
**nit** since your returned value cannot be an empty string (only possibly `undefined`) the `??` operator is better suited
```suggestion
        id: normalizePublicId(test.id) ?? '',
```

Reply:
I can't remember if i've ever used this operator before 😂